src/Integration/Telemetry/CFamilyTelemetryManager.cs

```diff
@@ -101,6 +96,41 @@ private void UpdateTelemetry()
             }
         }
 
+        private void CollectCMakeUsage()
+        {
+
+
+            var compilationDatabaseLocation = compilationDatabaseLocator.Locate();
+            var isCMakeAnalyzable = !string.IsNullOrEmpty(compilationDatabaseLocation);
+
+            if (isCMakeAnalyzable)
+            {
+                telemetryDataRepository.Data.CFamilyProjectTypes.IsCMakeAnalyzable = true;
+            }
+            else
+            {
+                telemetryDataRepository.Data.CFamilyProjectTypes.IsCMakeNonAnalyzable = true;
+            }
+
+            telemetryDataRepository.Save();
+        }
+
+        private void CollectVcxUsage()
+        {
+            var vcxProjectTypes = vcxProjectTypeIndicator.GetProjectTypes();
+
+            if (vcxProjectTypes.HasAnalyzableVcxProjects)
+            {
+                telemetryDataRepository.Data.CFamilyProjectTypes.IsVcxAnalyzable = true;
+            }
+            if (vcxProjectTypes.HasNonAnalyzableVcxProjects)
```

Comment:
In theory a solution can contain both analyzable and non analyzable, although it might not be a real use case in practice. Still, it's worth capturing.